### PR TITLE
20 doc drafting

### PR DESCRIPTION
**Pull request by EliotCassidy**

### CODE REVIEW

File: docs/connectors/source/application.md [hunk 1]
```diff
@@ -1,3 +1 @@
-# Application Source
-
-The Application Source Connector allows you to directly send data to Popsink using Python, Node, Go, Java or other programming languages. It is the most efficient way to build a direct integration between your application and Popsink. Start by creating a connector then you will be able to register topics/schemas to send data to using you application.
+The Application Source Connector allows you to directly send data to Popsink using Python, Node, Go, Java or other programming languages. It is the most efficient way to build a direct integration between your application and Popsink. Start by creating a connector then you will be able to register topics/schemas to send data to using you application.
```
Comment (Romderful) [inline docs/connectors/source/application.md:1]: ...to using _**your**_ application.

Comment (EliotCassidy) [reply docs/connectors/source/application.md:1]: Thanks :)

File: docs/connectors/source/attio.md [hunk 2]
```diff
@@ -0,0 +1 @@
+"The Attio Source Connector integrates with Attio, a modern customer relationship management (CRM) platform designed for data-driven teams. Attio offers a flexible and customizable approach to managing customer data, interactions, and relationships. It allows users to create custom objects, fields, and workflows to match their specific business processes. The connector enables real-time synchronization of data from Attio to your data pipeline, facilitating analytics, reporting, and integration with other business systems. This connector is particularly useful for companies looking to leverage their CRM data for advanced analytics, customer insights, and cross-functional collaboration.
```
[the remark this answers is not in the document]
Comment (EliotCassidy) [reply docs/connectors/source/attio.md:1]: Are you sure ?

File: docs/connectors/source/kafka.md [hunk 3]
```diff
@@ -1,3 +1 @@
-# Kafka Source
```
Comment (ggekos) [inline docs/connectors/source/kafka.md:1]: all the first title should be kept for SEO purpose

File: docs/connectors/source/attio.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Romderful) [reply docs/connectors/source/attio.md:1]: well, that's how i would write it haha. @bdjidi, @ggekos ? :)

**Pull request merged**